xarray/tests/test_backends.py

```diff
@@ -5009,7 +5008,7 @@ def test_use_cftime_standard_calendar_default_in_range(calendar):
 
     with create_tmp_file() as tmp_file:
         original.to_netcdf(tmp_file)
-        with pytest.warns(None) as record:
+        with warnings.catch_warnings(record=True) as record:
```

Comment:
These are not converted because they call `_assert_no_dates_out_of_range_warning` below - so this slightly relaxes condition (else it would not allow any warnings and later check if some specific warnings are not raised..., could alternatively just check that no warnings are raised).